src/sqlfluff/core/__init__.py

```diff
@@ -3,7 +3,7 @@
 # flake8: noqa: F401
 
 # Config objects
-#from .config import FluffConfig
+from .config import FluffConfig

```

Comment:
I think this had been commented out as a workaround for the dependency cycle. I was able to re-enable this code.